webviz_config/_docs/_build_docs.py

```diff
@@ -105,9 +104,9 @@ def get_plugin_documentation() -> defaultdict:
     """
 
     plugin_doc = [
-        _document_plugin(plugin)
-        for plugin in _get_webviz_plugins(webviz_config.plugins)
-        if not plugin[0].startswith("Example")
+        _document_plugin((plugin_name, plugin_registry.plugin_class(plugin_name)))
+        for plugin_name in plugin_registry.all_plugin_names()
+        if not plugin_name.startswith("Example")
```

Comment:
Must say: I wasn't aware of this, and not anything to do with the PR, but to skip plugins that starts with `Example` in the doc is quite hacky and magic 😋

Reply:
Maybe we (in a separate PR) can consider #381 and then remove `if not plugin_name.startswith("Example")` here afterwards?